all_subcommands.md

```diff
@@ -0,0 +1,674 @@
+                        Max lifetime in hours for this access token; the token will be refreshed by an asynchronous Rucio daemon. If not specified, refresh will be stopped after 4 days. This option is
+                        effective only if --oidc-scope includes offline_access scope for a refresh token to be granted to Rucio.
+  --oidc-issuer OIDC_ISSUER
+                        Defines which Identity Provider is going to be used. The issuer string must correspond to the keys configured in the /etc/idpsecrets.json auth server configuration file.
+  --certificate CERTIFICATE
+                        Client certificate file.
+  --ca-certificate CA_CERTIFICATE
+                        CA certificate to verify peer against (SSL).
+```
+# account 
+```
+usage: rucio account [-h] [--type {GROUP,USER,SERVICE}] [-a ACCOUNT] [--id IDENTITY] [--filters FILTERS] {ban,attribute,limit,identity,list,add,info,remove,usage,set} ...
+
+Default Operation: list
+Methods to add or change accounts for users, groups, and services. Used to assign privileges.
+Operations: ['list', 'add', 'info', 'remove', 'usage', 'set']
+Subcommands: ['ban', 'attribute', 'limit', 'identity']
+
+Usage Example:
+$ rucio account  # List all accounts on the instance$ rucio account add --account user_jdoe --type USER  # Create a new user account
+$ rucio account set --account user_jdoe --key email --value jdoe@cern.ch  # Update jdoe's email
+$ rucio account usage --account root  # Show all the usage history for the account root
+
+positional arguments:
+  {ban,attribute,limit,identity,list,add,info,remove,usage,set}
+    ban                 Disable an account. In case of accidental ban, use `$ rucio account ban remove`
+    attribute           Add additional key/value pairs associated with an account for organizational purposes.
+    limit               Manage storage limits for an account at a given RSE.
+    identity            Manage identities (DNs) on an account.
+    list                List accounts.
+    add                 Add a new account.
+    info                Get all stats on an account, including status, account type, and dates of creation and updates.
+    remove              Delete an account.
+    usage               See historical usage for an account
+    set                 Change the basic account settings.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --type {GROUP,USER,SERVICE}
+                        Account Type
+  -a ACCOUNT, --account ACCOUNT
+                        Account name
+  --id IDENTITY         Identity (e.g. DN)
+  --filters FILTERS     Filter arguments in form `key=value,another_key=next_value`
+```
+ ## account ban 
+```
+rucio account ban -h 
+usage: rucio account ban [-h] {add,remove} ...
+
+Default Operation: add
+Disable an account. In case of accidental ban, use `$ rucio account ban remove`
+Operations: ['add', 'remove']
+Subcommands: ['ban', 'attribute', 'limit', 'identity']
```

Comment:
This is a mistake - I need to go back and double check all subcommands don't show the subcommand menu